### ISSUE TITLE
Feature request: home screen widget

### ISSUE THREAD

**Issue by Yohe-Am** [Mosc/Glider#68]
Recently switched from over from Materialism after being wowed by Glider Ui/Ux and one thing I miss is the home screen widget. I find widgets are excellent for _feeds_ I passively _consume_. Can you please make this happen dev?

**Comment by Mosc** [comment]
Non-trivial for Flutter apps because native code is required to display widgets on the home screen, while accessing data from the Flutter side. [home_widget](https://pub.dev/packages/home_widget) can assist with this. A bit messy, but might look into this in the future.

**Comment by ndom91** [comment]
Commenting here to get notifications and show my support. Really wish i could replace materialistic widget with something from Glider ❤️
